charts/aws-ssm-agent-installer/values.yaml

```diff
@@ -10,3 +10,5 @@ tolerations:
     operator: Exists
   - effect: NoSchedule
     operator: Exists
+imagePullSecrets:
+  - cred
```

Comment:
Can you please remove this default value or comment it out?

Reply:
Yes, done.